subsys/net/ip/net_context.c

```diff
@@ -735,15 +735,15 @@ static inline int send_syn_ack(struct net_context *context,
 }
 
 static inline int send_ack(struct net_context *context,
```

Comment:
I think we could also remove the "inline" as that is not really needed here. Probably gcc will do it automatically but still. This could be a separate patch thou.